EIPS/eip-2135.md

```diff
@@ -0,0 +1,74 @@
+
+---
+eip: 2135
+title: Consumable Interface
+author: Zainan Victor Zhou (@xinbenlv)
+discussions-to: https://github.com/ethereum/EIPs/issues/2135
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-06-23
+requires (*optional): N/A
+replaces (*optional): N/A
+---
+
+
+ERC: Consumable Interface
+
+## Simple Summary
+An interface marking certain digital assets being consumable. 
+
+## Abstract
+The interface identifies functions and events needed for creating a contract to be able to mark a digital asset as "consumable", and react to the request of "consumption".
+
+## Motivation
```

Comment:
This motivation doesn't really explain **why** a digital asset needs to be 'consumed'. For example, to avoid having two people enter a concert with the same ticket.

Reply:
answered in https://github.com/ethereum/EIPs/pull/2171#issuecomment-510272832